openach/views/boards.py

```diff
@@ -50,11 +52,18 @@
 @account_required
 def board_listing(request):
     """Return a paginated board listing view showing all boards and their popularity."""
-    board_list = Board.objects.user_readable(request.user).order_by("-pub_date")
+    board_obj = Board.objects.user_readable(request.user).annotate(
+        board_title_lower=Lower("board_title")
```

Comment:
Please add a comment of why this query needs to be annotated -- where is the annotation actually being used? I searched for `board_title_lower` and didn't see anywhere